src/main/java/uk/gov/companieshouse/acsp/manage/users/mapper/AcspMembershipCollectionMappers.java

```diff
@@ -8,28 +8,16 @@
 import org.springframework.stereotype.Component;
 import uk.gov.companieshouse.acsp.manage.users.model.AcspDataDao;
 import uk.gov.companieshouse.acsp.manage.users.model.AcspMembersDao;
-import uk.gov.companieshouse.acsp.manage.users.service.AcspDataService;
-import uk.gov.companieshouse.acsp.manage.users.service.UsersService;
 import uk.gov.companieshouse.api.accounts.user.model.User;
 import uk.gov.companieshouse.api.acsp_manage_users.model.AcspMembership;
 import uk.gov.companieshouse.api.acsp_manage_users.model.AcspMembershipsList;
 import uk.gov.companieshouse.api.acsp_manage_users.model.AcspMembershipsListLinks;
 
 @Component
-public class AcspMembershipCollectionMappers {
-
-    private final AcspMembershipMapper baseMapper;
-    private final UsersService usersService;
-    private final AcspDataService acspDataService;
+public class AcspMembershipCollectionMappers extends AcspMembershipMapperImpl {
```

Comment:
why impl rather than the mapper interface or abstract class?.

Reply:
Thought it might make the code a little bit simpler to use Impl. I've changed it to use the interface